java/ql/src/Telemetry/APIUsage.qll

```diff
@@ -0,0 +1,44 @@
+import java
+private import semmle.code.java.dataflow.FlowSteps
+private import semmle.code.java.dataflow.ExternalFlow
+
+string jarName(CompilationUnit cu) {
+  result = cu.getParentContainer().toString().regexpCapture(".*/(.*\\.jar)/?.*", 1)
+}
+
+predicate isJavaRuntime(Callable call) {
+  jarName(call.getCompilationUnit()) = "rt.jar" or
+  call.getCompilationUnit().getParentContainer().toString().substring(0, 14) = "/modules/java."
```

Comment:
Do we need to special-case the Java standard libraries?

Reply:
Not anymore, folded the JDK query into the external API query so we don't need to differentiate between them anymore.